TrackerConditions/fcl/prolog.fcl

```diff
@@ -116,7 +116,8 @@ StrawElectronics : {
    TDCResolution : 0.1
    electronicsTimeDelay : 0.0
    eventWindowMarkerROCJitter : 0.5
-   flashEnd : 500.0
+   digitizationStart : 500.0
+   digitizationEnd : 1695.0
```

Comment:
Dont we need a different configuration for Offspill?  And how do we coordinate that to be consistent with the EWMProducer?  
in MDC2018, we digitized out to 1750 so we could see the start of the next proton pulse, which could be useful for studying sidebands in extinction.  Do we no longer have that option?

Reply:
I think for now the offspill digitization window length can actually just be the event length, that way it will automatically be kept in sync. I'll have to do small changes to the wrapping to get digitization windows out farther than 1695. The hardware requirement is just that the next marker arrives after the end of the window, so this says the marker has to be delayed relative to the beam by at least 55ns. They are planning on delaying it, I don't think there's any other requirements on how much so this should be perfectly fine.

Reply:
So digitizationStart,End are ignored offspill?  I missed that in the code.  maybe these parameters should be defined as onspillDigitizationStart,End?  Or could they be defined relative to the start and end of the EWM and used in both?  They should of course model what actually happens in hardware/firmware.

Reply:
Here too I'd feel more comfortable if the configuration for VST/commissioning were explicit, ie didn't require setting parameters that are never used.  Could you have a dedicated fcl bloc for VST/commissioning, where these are set to 'flag' values making it clear they are not intended to be used (ie the code will throw otherwise)?

Reply:
Perhaps we should step back.  Do we need a single point of maintenance in fcl to say "this job is VST"?  

There are several options.  Here is one that we have kicked around but have always deferred until something more concrete came along.

We have presumed that data would come with a header that describes what you need to know about the run time configuration and data type.  Is it time to prototype one of these?  If we want to go this way, should it be in the run/subrun/event?  Also if we go this way we need to invent the analog for MC events.

Reply:
My concern is that these are parameters that are set in the hardware in all cases, and presumably will be uploaded to the database for every run, and so we'd be making the database and the Offline proditions diverge there. And we already do have this marked in the data in a way by the offspill flag in the EventWindowMarker object.

Reply:
I presume that the DAQ will put a copy of the EventWindowMarker into the event as part of some sort of event header object.  Do we have that now?  We need to be able to see the EWM even after we have dropped all of the raw data.

Reply:
The EWT information will definitely be part of the event header. You'll need to ask Giani is this is already the case.